src/app/pages/form-validation/validation-list-item.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+import * as Colors from '@pxblue/colors';
+
+@Component({
+    selector: 'app-validation-list-item',
+    template: `
+        <mat-list-item style="height: auto;">
+            <mat-icon [style.color]="success ? Colors.blue[500] : Colors.gray[100]">{{ icon }}</mat-icon>
+            <span style="padding-left: 8px;">{{ validationMessage }}</span>
+        </mat-list-item>
+    `,
+})
+export class ValidationListItemComponent {
```

Comment:
I might call this something more descriptive like 'password-strength-check', but otherwise changes look good.